server/package.json

```diff
@@ -1,26 +1,26 @@
 {
-  "name": "@angular/language-server",
-  "description": "LSP server for Angular Language Service",
-  "version": "14.2.0-next.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/angular/vscode-ng-language-service.git"
-  },
-  "author": "Angular",
-  "license": "MIT",
-  "engines": {
-    "node": "^14.15.0 || >=16.10.0"
-  },
-  "bin": {
-    "ngserver": "./bin/ngserver"
-  },
-  "dependencies": {
-    "@angular/language-service": "14.2.0-next.0",
-    "vscode-jsonrpc": "6.0.0",
-    "vscode-languageserver": "7.0.0",
-    "vscode-uri": "3.0.3"
-  },
-  "publishConfig": {
-    "registry": "https://wombat-dressing-room.appspot.com"
-  }
+    "name": "@angular/language-server",
```

Comment:
Nit: I believe the 4 space indentation is accidental— generally it should remain two spaces as in other repos.